hnn_core/tests/test_dipole.py

```diff
@@ -48,6 +48,24 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # average an n_of_1 dipole list
+    single_dpl_avg = average_dipoles([dipole])
+    for dpl_key in single_dpl_avg.data.keys():
+        np.testing.assert_allclose(
+            dipole_read.data[dpl_key],
+            single_dpl_avg.data[dpl_key],
+            rtol=0,
+            atol=0.000051)
+
+    # average dipole list with one dipole object and a zero dipole object
+    n_times = len(dipole_read.data['agg'])
+    dpl_null = Dipole(np.zeros(n_times, ), np.zeros((n_times, 3)))
+    dpl_1 = [dipole, dpl_null]
+    dpl_avg = average_dipoles(dpl_1)
+    for dpl_key in dpl_avg.data.keys():
+        np.allclose(dpl_1[0].data[dpl_key],
```

Comment:
Maybe I am misunderstanding the code I think this should fail. Since you're averaging a `dipole` of random numbers with zeros, shouldn't every value be cut in half?

Reply:
And I see the tests passing so now I'm extra confused 😄

Reply:
@ntolley Yea, I see what you're talking about, I don't think it should be working either. I'll try debugging and see what's really happening.

Reply:
it's because `np.allclose` returns a bool. You need an assert statement to make it fail

Reply:
Thanks @jasmainak. I think I was able to fix it with your suggestion. Thanks @ntolley for bringing it to my attention.